src/pQueue.js

```diff
@@ -0,0 +1,39 @@
+class Node {
+  constructor(key, value) {
+    this.key = value
```

Comment:
you should store the priority in the Node & pass it into the enqueue function from the node itself